### PR TITLE
Dark Bootstrap theme, view JSON output in UI

### PR DESCRIPTION
**Pull request by buchowski**
- The info is now displayed after clicking the icon, instead of hovering over it.
- The logo has been removed. 
- The details button is smaller.
- Ticket info has been removed from the commits.

### CODE REVIEW

File: ui/app/index.html [hunk 1]
```diff
@@ -10,7 +10,8 @@
   <meta name="description" content="">
 
   <script src="bower_components/html5-boilerplate/dist/js/vendor/modernizr-2.8.3.min.js"></script>
-
+  <link rel="stylesheet" type="text/css" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
```
Comment (mihaitodor) [inline ui/app/index.html]: Shouldn't we vendor this file locally?

File: ui/app/services/services.js [hunk 2]
```diff
@@ -144,7 +154,7 @@ angular.module('sidecar.services', ['ngRoute', 'ui.bootstrap'])
 	stateService.waitFirstHaproxy.then(function() {
 		stateService.waitFirstServices.then(function() {
 			updateData();
-			$interval(updateData, 2000); // Update UI every 2 seconds
+			// $interval(updateData, 2000); // Update UI every 2 seconds
```
Comment (mihaitodor) [inline ui/app/services/services.js]: Why comment this out?

Comment (buchowski) [inline ui/app/services/services.js]: Good catch. I had commented this out for testing.

File: ui/app/index.html (hunk 1 above)
Comment (buchowski) [inline ui/app/index.html]: Sounds good to me. I'll store bootstrap locally instead of using the cdn.

**Pull request merged**